### PR TITLE
Ahodges/SD CA assessments

### PR DESCRIPTION
**Pull request by lamagnifica**
@aleffert @lou-wang @nasthagiri for the "Building and Running a Course" guide, I only plan on including this list of the supported problem types for mobile. A future story is planned to add best practices info as well.
@srpearce  more include tagging

### CODE REVIEW

File: en_us/shared/exercises_tools/Section_mobile_problems.rst [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+*********************************
+Mobile-Ready Problem Types
+*********************************
+
+Learners can read and submit answers for the following types of problems while they use the edX mobile app. 
+
+* :ref:`Checkbox`
+* :ref:`Dropdown`
+* :ref:`Math Expression Input`
```
Comment (BenjiLee) [inline en_us/shared/exercises_tools/Section_mobile_problems.rst]: I'm not sure if we support this(Math Expression Input). I could be wrong. I was under the assumption that we are supporting the basic assessments:
-Checkboxes
-Dropdown
-Multiple Choice
-Numerical Input
-Text Input

Comment (lamagnifica) [inline en_us/shared/exercises_tools/Section_mobile_problems.rst]: @BenjiLee who can I check this with?

Comment (BenjiLee) [inline en_us/shared/exercises_tools/Section_mobile_problems.rst]: @nasthagiri @aleffert @lou-wang

**Comment by lamagnifica** [comment]
@BenjiLee thank you -- correction made

**Comment by BenjiLee** [comment]
:+1:

**Pull request merged**